client/src/main/webjar/program-form-dialog/program-form-dialog.html

```diff
@@ -31,13 +31,18 @@
 
                     <!-- Public information -->
 
+                    <h2>
+                        <span md-colors="{color: 'primary'}">
+                            Organization information
+                        </span>
+                    </h2>
+
                     <md-input-container class="md-block" flex>
                         <label>
                             Company/Organization
                         </label>
                         <md-select name="organization"
                                    ng-model="programFormDialog.program.organization"
-                                   ng-model-options="{trackBy: '$value.id'}"
```

Comment:
This was causing undefined to be used to select to option with an undefined id, which is the new org.

That is why the field defaulted to new org instead of blank.